src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { Slider, Toggle } from '@flybywiresim/react-components';
-import { useSimVar } from '@instruments/common/simVars';
+import { useSimVar } from '../../Common/simVars';
```

Comment:
Please keep the @ import.

Reply:
For some reason with this specific import it fails when the code is compiled with `npm run serve:efb`. I see the same import has already been changed in some other files to what I have here. I'm not arguing against the change. I'm just explaining why I made the change. I will revert back to the original import which does work when building the whole project with `scripts/build.sh`

Reply:
> For some reason with this specific import it fails when the code is compiled with `npm run serve:efb`. I see the same import has already been changed in some other files to what I have here. I'm not arguing against the change. I'm just explaining why I made the change. I will revert back to the original import which does work when building the whole project with `scripts/build.sh` 

Very odd, in that case you can opt for what works :)

Reply:
No problem. I can change it again in future if it is still a problem. I changed back to `@instruments/common/simVars` already. I noticed that intellij/webstorm also suggests @instruments/common/simVars for the import.